OPEN_PROBLEMS/PUBSUB_AT_SCALE.md

```diff
@@ -0,0 +1,128 @@
+# PubSub at Scale
+
+## Short Description
+> In one sentence or paragraph.
+
+Publish/Subscribe (pubsub) messaging systems have been proposed and traditionally used to disintermediate senders and receivers of messages. That is, in pubsub systems, publishers of content do not send the published messages directly to one or a group of receivers, but instead *publishers are sending messages to a topic.* This enhances asynchronous communication and reduces tremendously network traffic and bandwidth-requirements.
+
+## Long Description
+
+Pubsub systems are a subgroup of broadcast trees, according to which, once messages are published, they are broadcast to all nodes in the tree. Although broadcast can ensure timely delivery of messages to all receipients, it causes severe stress to the system in terms of bandwidth needed to deliver the messages.
+
+One of the main benefits of pubsub systems is that receivers of information (or subscribers) can individually pull information when they need to. This reduces a lot the strain put in the system to deliver messages as they are published. However, in case of large-scale systems, the overhead of publishing information to a topic can still overload the system.
```

Comment:
Both the Short & Long Description are describing the system but give no context in what the Open Problem is. There must be an highlight of the challenges of building such system at the scale we need